packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -25,8 +101,20 @@ const Wrapper = styled.div`
     }
 `;
 
+const KeepWidth = styled.div<{ maxWidth?: string | undefined; border: boolean }>`
```

Comment:
What should this do? It looks like that it does not keep width because when you hit a cross button in e.g. Send section, there is some horizontal animation that resizes the bar.

Reply:
fixed in eb9750cc134ed0fb8ec75d435421489ad4f796fc